docs/data_access.rst

```diff
@@ -66,80 +128,141 @@ Datasette
+
+.. note::
+
+   You will need to install pandas with the
+   `extra cloud dependencies <https://pandas.pydata.org/pandas-docs/stable/getting_started/install.html#install-optional-dependencies>`__.
+
+.. code-block:: python
+
+   import pandas as pd
+
+   # Outputs from the most recent nightly build:
+   nightly_df = pd.read_parquet("s3://pudl.catalyst.coop/nightly/core_eia__codes_energy_sources.parquet")
+   # Outputs from the most recent stable data release:
+   stable_df = pd.read_parquet("s3://pudl.catalyst.coop/stable/core_eia__codes_energy_sources.parquet")
+   # A specific stable version of the data:
+   versioned_df = pd.read_parquet("s3://pudl.catalyst.coop/v2024.11.0/core_eia__codes_energy_sources.parquet")
+
+DuckDB
+^^^^^^
+
+Using `DuckDB <https://duckdb.org/2021/06/25/querying-parquet.html>`__
+and the `httpfs extension <https://duckdb.org/docs/guides/network_cloud_storage/s3_import.html>`__
+
+.. code-block:: sql
 
-If you don't want to mess with the API
-or CLI, you can also download the data directly over HTTPS. The download links for
-each table's Parquet file can be found in
-the :doc:`PUDL data dictionary page </data_dictionaries/pudl_db>`.
+   -- Install the httpfs extension once and it will be available in subsequent sessions
+   INSTALL httpfs;
+   SELECT * FROM read_parquet('s3://pudl.catalyst.coop/nightly/core_eia__codes_energy_sources.parquet');
 
-These are the download links for the PUDL and raw FERC SQLite databases:
+Other Dataframe Libraries
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Similar functionality exists for the `dplyr library in R
+<https://www.pmassicotte.com/posts/2024-05-01-query-s3-duckplyr/>`__, the `polars
+library in Rust <https://docs.pola.rs/user-guide/io/cloud-storage/>`__, and many other
+programmatic data analysis tools.
+
+The AWS CLI
+^^^^^^^^^^^
+
+You can also use `the AWS CLI <https://aws.amazon.com/cli/>`__ to see what data is
+available and download it locally. For example, to list the contents of the AWS S3
+bucket to see what historic versions are available, and then download the full PUDL
+SQLite database from the nightly build outputs:
+
+.. code-block:: bash
+
+   aws s3 ls --no-sign-request s3://pudl.catalyst.coop/
+   aws s3 cp --no-sign-request s3://pudl.catalyst.coop/nightly/pudl.sqlite.zip .
+
+The :doc:`PUDL data dictionary </data_dictionaries/pudl_db>` provides direct links to
+download the full Parquet output for every table alongside its metadata.
+
+The links below will download the most recent nightly builds of all the PUDL SQLite
+databases and their metadata in bulk.
 
 Fully Processed SQLite Databases
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 * `Main PUDL Database <https://s3.us-west-2.amazonaws.com/pudl.catalyst.coop/nightly/pudl.sqlite.zip>`__
 * `US Census DP1 Database (2010) <https://s3.us-west-2.amazonaws.com/pudl.catalyst.coop/nightly/censusdp1tract.sqlite.zip>`__
 
-Hourly Tables as Parquet
```

Comment:
Removed this section since we are now providing links to all Parquet tables in the Data Dictionary, and they're accessed the same as all the other parquet outputs.